src/components/services/EnvironmentService.js

```diff
@@ -52,6 +52,15 @@ export const getPortalLink = () => {
   return "https://portal.dev.demo.catena-x.net/";
 };
 
+export const getLogoutLink = () => {
+  const hostname = getHostname();
+  if (hostname === "country-risk-dashboard.dev.demo.catena-x.net")
```

Comment:
This can't be the right way on how to handle this.

This should be (in my opinion) be configurable from the helm chart itself and not hard coded.

Reply:
@fabiodmota 
as discussed in the teams chat, please refer to those as illustrative commits for the approach we used:
https://github.com/catenax-ng/tx-portal-frontend/pull/110/commits/1a990af74a6b071d7d1bafe3247fcda79efc3693
https://github.com/catenax-ng/tx-portal-frontend/pull/110/commits/b1d3301989ec35fa492dcf2ecdb9288146677fbb
Even tough implementation-wise it's not entirely the same, I was inspired with this approach by the traceability team.

Reply:
@fabiodmota 
Just to be clear, additionally to the commits above, you also need to enable the env vars in the helm chart.
Reference PR: https://github.com/catenax-ng/tx-portal-cd/pull/5

Reply:
Corrected @SebastianBezold , as I said the css would have already been approved and validated by the @evegufy/julia team, now it is also corrected for the hardcoded envs thank for that @evegufy

Reply:
Corrected @SebastianBezold , 

Since the new approach i upgraded the version to 1.1.0 

As I said the css would have already been approved and validated by the @evegufy/julia team, now it is also corrected for the hardcoded envs thanks for that @evegufy 

Also added the arq42 documentation approved by Alexander.AK.Keppler@bmw.de

Reply:
Hi @fabiodmota , I cannot find the docs review here. Can you please point me to it, or ask Alexander to document it here in this PR?
@evegufy can you confirm, that the styleguide requirements are now met? cannot find that approval here either.
Please keep these alignments/approvals transparent. For me and anyone else not being part in your private conversations, it is quite hard to follow

Reply:
Hi @SebastianBezold, I can confirm the arc42 documentation is up to date.

Reply:
> Hi @fabiodmota , I cannot find the docs review here. Can you please point me to it, or ask Alexander to document it here in this PR? @evegufy can you confirm, that the styleguide requirements are now met? cannot find that approval here either. Please keep these alignments/approvals transparent. For me and anyone else not being part in your private conversations, it is quite hard to follow

@SebastianBezold There is currently a review by Julia going on as part of this ticket: https://jira.catena-x.net/browse/BPDM2-469

Reply:
@jjeroch: Could you please give your styleguide approval for QG5 here as requested by @Siegfriedk during the QG5 review.

Reply:
Hi @evegufy @alexsilva-CGI  and @alexKeppler ,
thank you for the input, but please refrain from doing reviews or posting updates in tools, that are not open for the open source community. This makes this whole effort intransparent.

So for this last time, i'll work my way through the comments. For upcoming reviews, please do them transparently for everyone